packages/sdk/src/page-utils.ts

```diff
@@ -79,6 +79,7 @@ export const getStaticSiteMapXml = (pages: Pages, updatedAt: string) => {
   const allPages = [pages.homePage, ...pages.pages];
   return (
     allPages
+      .filter((page) => (page.meta.documentType ?? "html") === "html")
```

Comment:
I read this twice, more straight forward
```suggestion
      .filter((page) => page.meta.documentType  === "html" ||  page.meta.documentType === undefined)
```

Reply:
why? any metrics why 2nd is better?
if you don't like brakets it can be `"html" === page.meta.documentType ?? "html"` anyway for me current is much more readable vs `||`